android/assets/jsons/Civ V - Gods & Kings/TileImprovements.json

```diff
@@ -5,100 +5,90 @@
+            "Yields on average [+18 Gold] when pillaged"],
 		"shortcutKey": "F"
 	},
 	{
 		"name": "Lumber mill",
 		"terrainsCanBeBuiltOn": ["Forest"],
 		"production": 1,
 		"turnsToBuild": 7,
-        "pillageGold": 10,
 		"techRequired": "Construction",
-		"uniques": ["[+1 Production] <after discovering [Scientific Theory]>"],
+		"uniques": ["[+1 Production] <after discovering [Scientific Theory]>", "Yields on average [+10 Gold] when pillaged"],
 		"shortcutKey": "L"
 	},
 	{
 		"name": "Mine",
 		"terrainsCanBeBuiltOn": ["Hill"],
 		"production": 1,
 		"turnsToBuild": 7,
-        "pillageGold": 20,
 		"techRequired": "Mining",
-		"uniques": ["[+1 Production] <after discovering [Chemistry]>"],
+		"uniques": ["[+1 Production] <after discovering [Chemistry]>", "Yields on average [+20 Gold] when pillaged"],
 		"shortcutKey": "M"
 	},
 	{
 		"name": "Trading post",
 		"terrainsCanBeBuiltOn": ["Plains","Grassland","Desert","Hill","Tundra","Forest","Jungle","Flood plains"],
 		"gold": 1,
 		"turnsToBuild": 7,
-        "pillageGold": 20,
 		"techRequired": "Guilds",
-		"uniques": ["[+1 Gold] <after discovering [Economics]>"],
+		"uniques": ["[+1 Gold] <after discovering [Economics]>", "Yields on average [+20 Gold] when pillaged"],
 		"shortcutKey": "T"
 	},
 
 	// Resource-specific
 	{
 		"name": "Camp",
 		"turnsToBuild": 7,
-        "pillageGold": 10,
 		"techRequired": "Trapping",
-		"uniques": ["Does not need removal of [Forest]","Does not need removal of [Jungle]","[+1 Gold] <after discovering [Economics]>", "Can only be built to improve a resource"],
+		"uniques": ["Does not need removal of [Forest]","Does not need removal of [Jungle]","[+1 Gold] <after discovering [Economics]>", "Can only be built to improve a resource", "Yields on average [+10 Gold] when pillaged"],
 		"shortcutKey": "C"
 	},
 	{
 		"name": "Oil well",
 		"turnsToBuild": 9,
-        "pillageGold": 20,
 		"techRequired": "Biology",
-		"uniques": ["Can only be built to improve a resource", "Cannot be built on [Water] tiles"],
+		"uniques": ["Can only be built to improve a resource", "Cannot be built on [Water] tiles", "Yields on average [+20 Gold] when pillaged"],
 		"shortcutKey": "W"
 	},
 	{
 		"name": "Offshore Platform",
-        "pillageGold": 20,
 		"techRequired": "Refrigeration",
-		"uniques": ["Can only be built to improve a resource", "Can only be built on [Water] tiles"],
+		"uniques": ["Can only be built to improve a resource", "Can only be built on [Water] tiles", "Yields on average [+20 Gold] when pillaged"],
 		"shortcutKey": "P"
 	},
 	{
 		"name": "Pasture",
 		"turnsToBuild": 8,
-        "pillageGold": 10,
 		"techRequired": "Animal Husbandry",
-		"uniques": ["[+1 Food] <after discovering [Fertilizer]>", "Can only be built to improve a resource"],
+		"uniques": ["[+1 Food] <after discovering [Fertilizer]>", "Can only be built to improve a resource", "Yields on average [+10 Gold] when pillaged", "Yields [+10 Food, +10 Culture] when pillaged"],
```

Comment:
Is that your test? "Cul"-ture from manure?

Reply:
Yeah it was haha, you can see it in the screenshot in the OP. Who knows, maybe it's some artistic manure.